addon/components/cfb-form-editor/question/options.js

```diff
@@ -83,6 +101,52 @@ export default RenderComponent.extend({
     this.setDirty();
   },
 
+  setupUIkit() {
+    UIkit.util.on("#option-list", "moved", (...args) =>
+      run(this, this._handleMoved, ...args)
+    );
+  },
+
+  _handleMoved({ detail: [sortable] }) {
+    const options = [...sortable.$el.children];
+
+    this.reorderQuestions.perform(
+      options.map((option) =>
+        addQuestionPrefix(option.firstElementChild.id, this.questionSlug)
+      )
+    );
+  },
+
+  reorderQuestions: task(function* (slugs) {
+    try {
+      slugs.pop();
+
+      yield this.apollo.mutate({
+        mutation: TYPES[this.model.__typename],
+        variables: {
+          input: {
+            slug: this.questionSlug,
+            label: this.model.label,
+            options: slugs,
+            clientMutationId: v4(),
```

Comment:
Out of curiosity: I see that we're setting `clientMutationId` also in other mutations - why? I thought this was optional.

Reply:
Isn't necessary.. Drop it.

Reply:
Removed it